src/inversify.config.ts

```diff
@@ -1,14 +1,19 @@
 import { Container } from 'inversify';
+import * as express from 'express';
 import { UserService } from './modules/user/user.service.ts';
 import TYPES from './constant/types.ts';
 import { TodoService } from './modules/todo/todo.service.ts';
 import { AuthService } from './modules/auth/auth.service.ts';
+import { JwtProtect } from './middleware/protect.ts';
 
 const APIContainer = new Container();
 APIContainer.bind<UserService>(TYPES.UserService).to(UserService);
 APIContainer.bind<TodoService>(TYPES.TodoService).to(TodoService);
 APIContainer.bind<AuthService>(TYPES.AuthService).to(AuthService);
 
-//APIContainer.bind<UserService>(TYPES.UserService).to(AuthService);
+//Auth middleware
+APIContainer.bind<express.RequestHandler>('AuthMiddleware').toConstantValue(
```

Comment:
'AuthMiddleware' should be saved in `src/constant/types.ts`.
It's better not to use string as identified as it's easier to make a mistake this way.